travis/install_custom_pg

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eux
+
+# clone PostgreSQL 
+cd ~
+git clone -b REL${PGVERSION//./_}_STABLE --depth 1 git://git.postgresql.org/git/postgresql.git
+
+# we will use this to parallelize PostgreSQL compilation
+procs="$(nproc)"
+mjobs="$(expr $procs + 1)"
+
+# configure, build and install PostgreSQL
+cd postgresql
+./configure --enable-cassert --enable-debug --with-openssl
```

Comment:
Add `--bindir="/usr/lib/postgresql/${PGVERSION}/bin"` to the `configure` arguments so you can just have the binaries live in the same spot whether PGDG or custom.